material-overrides/assets/stylesheets/home.css

```diff
@@ -7,8 +7,29 @@
   --min-padding: 0 3em;
 }
 
+:root,
+[data-md-color-scheme="default"] {
+  /* Home cards */
+  --home-card-fg-color: var(--storm-gray-950);
+  --home-card-bg-color: var(--white);
+  --home-card-border-color: var(--storm-gray-200);
+
+  /* Home buttons */
+  --home-button-border-color: var(--black);
```

Comment:
This seems unnecessary, we can just use `button-border-color` defined in the polkadot.css file